README.adoc

```diff
@@ -1,5 +1,21 @@
+:projectKey: org.optaplanner:optaplanner
+:sonarBadge: image:https://sonarcloud.io/api/project_badges/measure?project={projectKey}
+:sonarLink: link="https://sonarcloud.io/dashboard?id={projectKey}"
+
 = OptaPlanner
 
+image:https://img.shields.io/badge/stackoverflow-ask_question-orange.svg?logo=stackoverflow[
+"Ask question on Stack Overflow", link="https://stackoverflow.com/questions/tagged/optaplanner"]
+image:https://img.shields.io/badge/zulip-join_chat-brightgreen.svg?logo=zulip[
+"Join Zulip Chat", link="https://kie.zulipchat.com/#narrow/stream/232679-optaplanner"]
+
+{sonarBadge}&metric=alert_status["Quality Gate Status", {sonarLink}]
+{sonarBadge}&metric=reliability_rating["Reliability Rating", {sonarLink}]
+{sonarBadge}&metric=security_rating["Security Rating", {sonarLink}]
+{sonarBadge}&metric=sqale_rating["Maintainability Rating", {sonarLink}]
+{sonarBadge}&metric=ncloc["Lines of Code", {sonarLink}]
+{sonarBadge}&metric=coverage["Coverage", {sonarLink}]
+
 https://www.optaplanner.org/[www.optaplanner.org]
```

Comment:
@yurloc Already merged, but can you move the optaplanner.org link before the badges?

Reply:
Let's do the "Ask a question" and zulip link below the sonar badges? Quality comes before get help in a "getting started" journey.

Reply:
Let me try editing that directly

Reply:
Already moved optaplanner.org link. Please rebase.

Yes, please fix directly as you like.